rasa/server.py

```diff
@@ -26,14 +24,17 @@
 from rasa.core import broker
```

Comment:
I think we have to add locks to `execute_action`, `add_message`, `replace_events`, `append_events` as well.